src/main/java/appeng/parts/reporting/AbstractMonitorPart.java

```diff
@@ -63,6 +63,7 @@ public abstract class AbstractMonitorPart extends AbstractDisplayPart
     @Nullable
     private AEKey configuredItem;
     private long amount;
+    private boolean canCraft;
```

Comment:
This needs to be moved to `ConversionMonitorPart` imo, no need for it in the superclass.